examples/fakegroundstation/ground_station_service.py

```diff
@@ -0,0 +1,136 @@
+import time
+from concurrent import futures
+
+import grpc
+from google.protobuf.timestamp_pb2 import Timestamp
+
+from stellarstation.api.v1.groundstation import groundstation_pb2_grpc
+from stellarstation.api.v1.groundstation import groundstation_pb2
+
+
+SECONDS_BEFORE_PLAN_START = 10
+PLAN_DURATION_SECONDS = 600
+CURRENT_PLAN_ID = "3"
+
+
+class GroundStationServiceServicer(groundstation_pb2_grpc.GroundStationServiceServicer):
+    def ListPlans(self, request, context) -> groundstation_pb2.ListPlansResponse:
+        """Lists the plans for a particular ground station.
+
+        The request will be closed with an `INVALID_ARGUMENT` status if `ground_station_id`,
+        `aos_after`, or `aos_before` are missing, or the duration between the two times is longer than
+        31 days.
+        """
+        print('Got request for ListPlans')
+        if request.ground_station_id is None or request.ground_station_id == "":
```

Comment:
You could write `if not request.ground_station_id:` instead to check if the string is empty.